Makefile

```diff
@@ -134,12 +130,6 @@ build-app-for-simulator:
 # ----------------------------------------------------------------------------------------------------------------------
 # Devices - SDK
 
-USB_DEVICE_ID := $(strip $(shell system_profiler SPUSBDataType | sed -n -E -e "/(iPhone|iPad)/,/Serial/s/ *Serial Number: *(.+)/\1/p" ))
-
-ifneq ($(USB_DEVICE_ID),)
-USB_DEVICE_DESTINATION := -destination 'platform=ios,id=$(USB_DEVICE_ID)'
-endif
-
 # Xcode build command for building for device. The CODE_SIGNING_* variables are so that no code signing occurs on CI -
 # this is because AWS Device Farm re-signs the applications. This may need to change if a different provider is used.
```

Comment:
AWS Device Farm mentioned here as well, as well as on line 198(if that's even a problem).

Reply:
I'd like to keep it as it describes the exact behaviour of Device Farm. I'm not sure are we safe to remove all code signing part for now as we now do codesign for Test Lab.